Gemfile

```diff
@@ -46,12 +46,14 @@ group :development, :test do
   gem 'debug', platforms: %i[mri mingw x64_mingw]
   gem 'factory_bot_rails', '~> 6.2'
   gem 'faker', '~> 2.21'
+  gem 'rspec_api_documentation'
   gem 'rspec-rails', '~> 5.1', '>= 5.1.2'
 end
 
 group :development do
   # Use console on exceptions pages [https://github.com/rails/web-console]
   gem 'annotate', '~> 3.2'
+  gem 'apitome', '~> 0.3.1'
```

Comment:
Looks like tests are failing because this gem is not loading in the test environment

Reply:
You can prevent Apitome from loading [like this](https://github.com/rootstrap/forecast/blob/8f97bcfe9795cdde6a030b535a6998e86f86183a/config/initializers/apitome.rb#L1)